tests/test_iotedgedev.py

```diff
@@ -492,3 +496,16 @@ def test_push_modules_to_local_registry(prepare_solution_with_env):
             remove_docker_container("registry")
         if "registry" in get_all_docker_images():
             remove_docker_image("registry:2")
+
+# # TODO: The output of docker build logs is not captured by pytest, need to capture this before enable this test
```

Comment:
A TODO should also have a task in GitHub - Can you add if not there already?

Reply:
Added #405 to track this TODO item